### ISSUE TITLE
a11y: Color Contrast - Namespace Filter

### ISSUE THREAD

**Issue by rak-phillip** [rancher/dashboard#13461]
Insufficient color contrast for standard text

## Instances

"Foreground color: #3D98D3
Background color: #FFFFFF
Contrast ratio: 3.16:1
Header Area 
- [ ] All Namespaces (selected option in dropdown) 

Foreground color: #FFFFFF
Background color: #3D98D3
Contrast ratio: 3.16:1
Header Area 
- [ ] All Namespaces, Only User Namespaces, Only System Namespaces, Only Namespaced Resources, Only Cluster Resources(on hover and keyboard focus) 

Foreground color: #FFFFFF
Background color: #297DB4
Contrast ratio: 4.48:1
Header Area 
-  [ ] Starred, Cluster, Workloads, Apps, Service Discovery and so on navigation links (in selected state) "

## Actual result

The standard text with background/foreground color combinations having an insufficient contrast ratio making it difficult for people with low vision and color blindness to read standard text elements.

## Expected result

The contrast ratio between foreground text and background for the standard text should be sufficient.

## Recommendation

On all pages, ensure that each foreground/background color combination presents a contrast ratio of at least 4.5:1 for standard text.

### Light

![Image](https://github.com/user-attachments/assets/064b75c7-df1c-4b58-b3db-608e149f29aa)

![Image](https://github.com/user-attachments/assets/4a24c04a-6a37-49ac-8bc4-50658aded28a)

### Dark

![Image](https://github.com/user-attachments/assets/29c3a33c-75e7-45bb-8310-1537dfab483c)

**Comment by rak-phillip** [comment]
Marked as resolved in the VPAT. Issue 36
